packages/validator/templates/OAS2OIS.json

```diff
@@ -0,0 +1,288 @@
+					},
+					{
+						"__insert": {
+							"__target": "endpoints[_].fixedOperationParameters",
+							"__value": []
+						}
+					}
+				],
+				"parameters": {
+					"__arrayItem": {
+						"name": {
+							"__actions": [
+								{
+									"__copy": {
+										"__target": "apiSpecifications.paths.{{1}}.{{2}}.{{3}}.name"
+									}
+								},
+								{
+									"__copy": {
+										"__target": "endpoints[_].parameters[].name"
+									}
+								},
+								{
+									"__copy": {
+										"__target": "endpoints[_].parameters[_].operationParameter.name"
+									}
+								}
+							]
+						},
+						"in": {
+							"__regexp": "^(path|query|header|cookie)$",
+							"__actions": [
+								{
+									"__copy": {
+										"__target": "apiSpecifications.paths.{{1}}.{{2}}.{{3}}.in"
+									}
+								},
+								{
+									"__copy": {
+										"__target": "endpoints[_].parameters[_].operationParameter.in"
+									}
+								}
+							]
+						},
+						"schema": {
+							"__conditions": [
+								{
+									"__if": {
+										"default": ".*"
+									},
+									"__then": {
+										"default": {
+											"__actions": [
+												{
+													"__copy": {
+														"__target": "endpoints[_].parameters[_].default"
+													}
+												}
+											]
+										}
+									}
+								}
+							],
+							"__ignore": {}
+						},
+						"__ignore": {}
+					}
+				},
+				"__ignore": {}
+			}
+		}
+	},
+	"components": {
+		"schemas": {
+			"securitySchemes": {
+				"__objectItem": {
+					"type": {
+						"__regexp": "^(apiKey|http)$"
+					},
+					"in": {
+						"__regexp": "^(query|header|cookie)$"
+					},
+					"__conditions": [
+						{
+							"__if": {
+								"type": "^(apiKey|http)$"
+							},
+							"__then": {
+								"type": {
+									"__actions": [
+										{
+											"__copy": {
+												"__target": "apiSpecifications.components.securitySchemes.{{3}}.type"
+											}
+										}
+									]
+								}
+							}
+						},
+						{
+							"__if": {
+								"in": "^(query|header|cookie)$"
+							},
+							"__then": {
+								"in": {
+									"__actions": [
+										{
+											"__copy": {
+												"__target": "apiSpecifications.components.securitySchemes.{{3}}.in"
+											}
+										}
+									]
+								}
+							}
+						},
+						{
+							"__if": {
+								"type": "^apiKey$"
+							},
+							"__then": {
+								"name": {
+									"__actions": [
+										{
+											"__copy": {
+												"__target": "apiSpecifications.components.securitySchemes.{{3}}.name"
+											}
+										},
+										{
+											"__insert": {
+												"__target": "apiSpecifications.security.{{3}}",
+												"__value": []
+											}
+										}
+									]
+								}
+							}
+						},
+						{
+							"__if": {
+								"type": "^http$"
+							},
+							"__then": {
+								"scheme": {
+									"__regexp": "^(Basic|Bearer)$",
```

Comment:
Looks like OAS uses lower case: https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#basic-authentication-sample

Reply:
I was going to say that https://www.iana.org/assignments/http-authschemes/http-authschemes.xhtml starts with uppercase so we did the same but Andre actually used lower case https://github.com/api3dao/airnode/blob/116450562b7ee39134af78afaf09ad8af2478f5b/packages/ois/src/types.ts#L31

I'll update the docs, let's use lowercase

Reply:
@bbenligiray should this be lowercase only in OAS or does it apply to all specs, like [in API specs](https://github.com/api3dao/airnode/blob/1b7124caa2d44b555d360f5f152f5260663b8821/packages/validator/templates/apiSpecifications.json#L67) for example?

Reply:
It should be lowercase everywhere

Reply:
added to #197